.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Main
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container:
+      image: rust:1.71.1
```

Comment:
I think we should use MSRV (minimum supported Rust version) for CI checks

Reply:
The `near-sdk-rs` one? Sure.

Reply:
I would define its own MSRV for near-account-id. I feel we are not going to bump it much (ever?)